### DISCUSSION TITLE
Introductory tutorials

### DISCUSSION THREAD

**Discussion by VivienFuhrer** [ProxeusApp/community#6] [Q&A] [answered]
What kind of tutorials would be best for the use of Proxeus?
We already have the [User handbook](https://github.com/ProxeusApp/community/blob/master/handbook/handbook.md).
Maybe a video would be helpful. What is your opinion?

**Comment by loleg** [comment, marked as answer]
Update 2023: a screencast by @Ibrahim-Halil-Kuray is now part of the documentation and is visible at the [Proxeus Handbook](https://doc.proxeus.org/#/handbook)

Here are some links to introductory tutorials:

- [Proxeus White Paper](https://docs.google.com/document/d/1TTSTjRdZ3VMPVAV8_So6ycnKHqX3_Uos5txhcFAqiw4/edit)
- [Template Handbook](https://docs.google.com/document/d/1-vJsTrU3w8dEcDr3-nV5owtxqHWSjzEf2uk6m9-cMIs/preview)
- [Guide - Education Diploma](https://docs.google.com/document/d/1Gl6R1t0LYRK6kARScx5-vqpiKtd6xqcH2yPPC3wWMr4/preview)
- [Guide - Crypto-tax Workflow](https://docs.google.com/document/d/1lllYwzMWNESZg9vnOajssuxWaalMycwI2Zs8BmDwf1M/preview)
- [Guide - Proof of Existence](https://docs.google.com/document/d/19Q0w6vDDkQ3bQKjfMf-ksMNoGWjo-uxiMfskm2JHMqw/preview)
